src/components/DevelopmentGuide/DesignTokens.stories.mdx

```diff
@@ -0,0 +1,536 @@
+## Why Use Design Tokens?
+
+_"With design tokens, you can capture low-level values and then use them to
+create the styles for your product or app. You can maintain a scalable and
+consistent visual system for UI development."_ - Jina Bolton, Lead Designer for
+the Salesforce Design System
+
+Design tokens are a single source of truth when it comes to styles. They allow
+an organization to more easily build multiple applications - web and/or native -
+that feel and look the same without slowing down the development team.
+
+## How to Use the Reservoir Design Tokens
+
+The NYPL design tokens are built into the Reservoir Design System (DS) and are
+offered in two formats for consumption: javascript theme objects (recommended)
+and CSS variables. To access either of these formats, the DS`v0.25.9` or higher
+needs to be properly installed in a consuming React application.
+
+## Using Reservoir Design Tokens in Javascript
+
+It is recommended that consuming apps use javascript (i.e. CSS-in-JS) to apply
+styles to UI elements. Additionally, consuming apps should, as much as possible,
+use DS and native Chakra components. With that, apps will be able to utilize the
+[javascript design tokens](#reservoir-design-token-categories).
+
+To use the DS javascript design tokens, the`DSProvider` wrapper component must
+be imported and implemented properly in a consuming app. Please refer to the
+`DSProvider` section of the [DS Chakra
+UI](?path=/story/chakra-ui--page#dsprovider) page for instructions on how to
+implement the`DSProvider` component in a consuming app. Once this has been
+completed, javascript design tokens can be used to style DS and Chakra
+components.
+
+#### Design Tokens for DS and Chakra Components
+
+**IMPORTANT:** The general rule of thumb is that the styles and functionality of
+the DS components should not be altered. The components should be used with the
+styles and variants as they are depicted in Storybook. There may be times,
+however, when a component does require a modification. That being said, if a DS
+component does need a style change in a consuming app, designers/developers
+should first contact the Reservoir and UX teams about the change before it is
+implemented.
+
+The javascript design tokens can be used to populate the values for style props
+added directly to the components. The shorthard nature of the javascript tokens
+make it easy to quickly and logically add styles to a DS component.
+
+For example, if a`Heading` component needs to sit within a wrapper element
+and render its text using Reservoir's`brand primary` color and use a`light`
+font weight, the following can be done:
+
+```tsx
+<Box
+  backgroundColor="ui.bg.default"
+  border="1px solid"
+  borderColor="ui.border.default"
+  padding="inset.default"
+>
+  <Heading color="brand.primary" fontWeight="light">
+    Get a Digital Library Card Today in a Few Easy Steps
+  </Heading>
+</Box>
+```
+
+#### Design Tokens for Custom Components and Native HTML Elements
```

Comment:
Yes, this can be confusing.  This section talks about implementing design tokens in Javascript.  It is not about implementing javascript design tokens.  We are showing CSS variables being used, but we are using CSS variables in Javascript.  Let's sit on this for now, but we should ask others to chime in soon.